kinto/__main__.py

```diff
@@ -9,7 +9,7 @@
 
 from kinto.config import init
 
-CONFIG_FILE = 'config/kinto.ini'
+CONFIG_FILE = 'kinto/config/kinto.ini'
```

Comment:
so that the kinto.ini file is created within the kinto/config folder( where kinto.tpl is present) . If only config/kinto.ini is written, it creates a new config folder in the root directory and kinto.ini is created inside that.